srcpkgs/kvantum/template

```diff
@@ -1,21 +1,35 @@
 # Template file for 'kvantum'
 pkgname=kvantum
-version=1.0.5
+version=1.0.10
 revision=1
 build_wrksrc=Kvantum
-build_style=qmake
-hostmakedepends="qt5-tools qt5-qmake qt5-host-tools"
-makedepends="qt5-devel qt5-svg-devel qt5-x11extras-devel
- kwindowsystem-devel"
-short_desc="SVG-based theme engine for Qt4/Qt5, KDE and LXQt"
+build_style=cmake
+hostmakedepends="qt5-tools-devel qt5-host-tools qt6-tools-devel"
+makedepends="qt5-devel qt6-base-devel qt5-svg-devel qt6-svg-devel
+ qt5-x11extras-devel kwindowsystem-devel"
+short_desc="SVG-based theme engine for Qt4/Qt5/Qt6, KDE and LXQt"
```

Comment:
You can remove the mention of `Qt4` since this isn't packaged.

Looks good to me otherwise.